docs/configuration/outbox-event-router.asciidoc

```diff
@@ -0,0 +1,195 @@
+    "table": "outboxevent",
+    "snapshot": false,
+    "last_snapshot_record": null,
+    "xmin": null
+  },
+  "op": "c",
+  "ts_ms": 1556890294484
+}
+----
+
+This result was achieved with the link:#default-table-columns[Default Configuration] which assumes a table structure and event routing based on aggregates, in case you want a custom behavior, the smt is fully configurable, check the link:#configuration-options[Available configuration options].
+
+== Configuration
+
+=== Configuration options
+[cols="30%a,10%a,10%a,50%a",width=100,options="header,footer",role="table table-bordered table-striped"]
+|=======================
+|Property
+|Default
+|Group
+|Description
+
+|`table.field.event.id`
+|`id`
+|Table
+|The column which contains the Event ID within the outbox table
+
+|`table.field.event.key`
+|`aggregateid`
+|Table
+|The column which contains the Event Key within the outbox table, when this is set the value of this column will be used as a kafka message key
+
+|`table.field.event.type`
+|`type`
+|Table
+|The column which contains the Event Type within the outbox table
+
+|`table.field.event.timestamp`
+|
+|Table
+|Optionally you can override the Kafka message timestamp with a value from a chosen field, otherwise it'll be the debezium event processed timestamp.
+
+|`table.field.event.payload`
+|`payload`
+|Table
+|The column which contains the Event Payload within the outbox table
+
+|`table.field.event.payload.id`
+|`aggregateid`
+|Table
+|The column which contains the Payload ID within the outbox table
+
+|`table.fields.additional.placement`
+|
+|Table, Envelope
+|Extra fields can be added as part of the event envelope or a message header, format is a list of colon-delimited pairs or trios when you desire to have aliases, e.g. `id:header,field_name:envelope:alias`. Placement options are `header` and `envelope`
+
+|`table.fields.schema.version`
+|
+|Table, Schema
+|When set, it'll be used as Schema version as in the https://kafka.apache.org/20/javadoc/org/apache/kafka/connect/data/ConnectSchema.html#version--[Kafka Connect Schema] javadoc
+
+|`route.by.field`
+|`aggregatetype`
+|Router
+|The column which determines how the events will be routed, the value will become part of the topic name
+
+|`route.topic.regex`
+|`(?<routedByValue>.*)`
+|Router
+|The default regex to use within the RegexRouter, the default capture will allow to replace the routed field into a new topic name defined in `route.topic.replacement`
+
+|`route.topic.replacement`
+|`outbox.event.${routedByValue}`
+|Router
+|The name of the topic in which the events will be routed, a replacement `${routedByValue}` is available which is the value of The column configured via `route.by.field`
+
+|`debezium.op.invalid.behavior`
+|`warn`
+|Debezium
+|While Debezium is monitoring the table, it's expecting only to see 'create' row events, in case something else is processed this transform can log it as warning, error or stop the process. Options are `warn`, `error` and `fatal`
```

Comment:
I think DELETEs are to be expected as well, and they should silently be ignored. Only UPDATEs should never occur really.

Reply:
You're right, I've rephrased it!